packages/terra-popup/src/Popup.jsx

```diff
@@ -84,64 +112,92 @@ const defaultProps = {
   contentHeight: '80',
   contentWidth: '240',
```

Comment:
This might be non-passive so not a change we can make. But I would expect the default behavior of these to be auto / dynamic

Reply:
It would be non-passive, and in general I don't know if we want to encourage the consumption of the least perfomant option.